src/main/java/redis/clients/jedis/Pipeline.java

```diff
@@ -166,4 +166,9 @@ public void close() {
     return getResponse(BuilderFactory.STRING);
   }
 
+  public Response<Long> waitReplicas(int replicas, long timeout) {
```

Comment:
Do we need to declare this method in `RedisPipeline.java` or `BinaryRedisPipeline.java`? And add `@Override` to this method.